src/LocalRegistry.jl

```diff
@@ -346,10 +346,16 @@ end
 
 function get_remote_repo(package_path, gitconfig)
     git = gitcmd(package_path, gitconfig)
-    remote_name = split(readchomp(`$git remote`), '\n')
-    length(remote_name) > 1 && error("Repository has multiple remotes.")
-    remote_name[1] == "" && error("Repository does not have a remote.")
-    return readchomp(`$git remote get-url $(remote_name[1])`)
+    remote_names = split(readchomp(`$git remote`), '\n')
+    repos = String[]
+    for remote_name in remote_names
+        if !isequal(remote_name,"")
```

Comment:
Actually this is also useless as an empty string would only appear if there were no remotes at all. And in fact the `split` should be made with `keepempty = false` so that doesn't happen either. I think all of this can basically be collapsed to the original code with more helpful error messages.

Reply:
That makes sense. I wasn't sure if `git remote` would ever return something like `["Repo", ""]`. If the only time it would return `""` is with `[""]`, then `keepempty = false` seems like the right thing.

Reply:
I removed that check. The only purpose of `repos` is to provide a friendlier error message. If you don't want that, no worries. Feel free to close the PR. I have what I need.